src/sqlfluff/core/templaters/jinja.py

```diff
@@ -212,13 +214,16 @@ def _get_jinja_env(self, config=None):
             loader=FileSystemLoader(macros_path) if macros_path else None,
         )
 
-    def _get_macros_path(self, config):
-        macros_path = None
+    def _get_macros_path(self, config: FluffConfig) -> Optional[List[str]]:
         if config:
             macros_path = config.get_section(
                 (self.templater_selector, self.name, "load_macros_from_path")
             )
-        return macros_path
+            if macros_path:
+                result = [s.strip() for s in macros_path.split(",") if s.strip()]  # type: ignore
```

Comment:
This mypy error is due to some dodgy typing in `FluffConfig.get_section`:
![image](https://user-images.githubusercontent.com/80432516/150508412-7a28147f-cf7a-4a16-af18-95248cbb4408.png)

That return type should really just be `Any` so could you update that and remove the type ignore here

Reply:
Done!